bayesian/utils.py

```diff
@@ -410,87 +411,123 @@ class CveByDateEcosystemUtils:
     """
 
     # Get CVEs information by date
-    cve_nodes_by_date_script_template = """\
-    g.V().has('modified_date', modified_date)\
-    .has('vertex_label','CVE').as('cve')\
-    .coalesce(\
-    inE('has_cve').outV().as('epv').select('cve','epv').by(valueMap()),\
-    select('cve').by(valueMap()))\
+    cve_epv_nodes_by_date_script_template = """\
+    g.V().has('cve_id', cve_id)\
+    .in("has_cve").valueMap()\
     """
 
     # Get CVEs by date & ecosystem
     cve_nodes_by_date_ecosystem_script_template = """\
     g.V().has('modified_date', modified_date)\
-    .has('vertex_label','CVE')\
-    .has('ecosystem',ecosystem).as('cve')\
-    .coalesce(\
-    inE('has_cve').outV().as('epv').select('cve','epv').by(valueMap()),\
-    select('cve').by(valueMap()))\
+    .has('cecosystem',ecosystem)\
+    .has('cve_sources',cve_sources)\
+    .valueMap()\
     """
 
-    def __init__(self, bydate, ecosystem=None):
-        """Constructor."""
-        self._bydate = bydate
-        self._ecosystem = ecosystem
+    cve_nodes_by_date_ecosystem_script_template_all = """\
```

Comment:
If this template to get all EPVs, then why are you fetching by cecosystem field here?

Reply:
Here, I mean all EPVs in a particular ecosystem not filtered by different cve_sources. The other query named "cve_nodes_by_date_ecosystem_script_template" is filtered by the cve sources.